Readme draft

C-Stein opened the pull request:
First draft complete and ready for some feedback.

chortlehoort commented:
I played around with tables and it looks like Github adds to styling to them out of the box.  Looks pretty good.  Also made some editorial additions.

```
# Mastery Learning

Welcome to an experiment in education. Learning software development isn't ike learning anything else , so we're not going to treat it like anything else. This is likely going to be a very different experience from your high school chemistry class, or your college intro to philosophy.

## How is mastery learning different from what I've done before?
Basically, we're aiming for 100% of students to have 100% mastery of the material. This is an ambitous plan and we're plannning on reaching it by allowing students to work at their own pace, providing students with lots of instructor guidance, and generally being awesome.


## Differences between traditional education and mastery learning

<table>
<tr>
<th><h4>Mastery Learning</h4></th>
<th><h4>Traditional Education</h4></th>
</tr>
<tr>
  <td>
  Individualized progress and flexible class timing aimed at 100% of students achieving 100% mastery
  </td>
  <td>
  Learning on a schedule with a syllabus, regardless of student understanding
  </td>
</tr>

<tr>
  <td>
  Students work through material at their own pace
  </td>
  <td>
  All students work on the same material at the same time
  </td>
</tr>

<tr>
  <td>
  Instructor will make sure that all material has been  presented by the end of the class, but individual students are responsible for their own progress through the material
  </td>
  <td>
  Teacher takes responsibility for keeping the class on schedule and for finishing the material by the end of the course
  </td>
</tr>

</table>

## What responsibilities are on the instructor?
1. The instructor will present all of the critical material necessary to build a capstone by the end of the course.
1. The instructor will make sure all student questions are answered. Even the dumb ones, because dumb questions are often the most enlightening.


## What responsibilities are on the student?

1. Do good student stuff
  * Show up to class 
  * Participate in class discussions and group projects 
  * Complete the exercises for each  milestone, the quiz for each milestone, and top it all off with a capstone project 
  * Timing is flexible, if  you need a little more time to completely understnad a topic, take some time, ask for help and fill in any gaps in your understanding.
1. Ask questions! Even if the question is a "huh, I don't know where to start," it is absolutely imperative that students take responsibility for their own understanding and clarify any questions right away.
1. Make sure you are putting in the time and effort to reach success.
1. Work through the exercises for each milestone before you attempt the quiz. Skipping ahead to the quiz is the best way to cheat yourself out of understanding.

> **Bottom Line:** We will give you all of the ingredients, but it's up to you to bake the cake.

###So, why do things this way?
 First, and most importantly, in software development, each concept builds upon the previous concept, so absolute mastery of each concept is necessary before moving on. Second, teaching adults over the course of 6 months is a thing and the adults we accept into the program are expected to have the work ethic needed to forge their own path.

For a more in-depth vision of mastery learning at NSS you can read the article [An Alternative for Teaching Software Development](http://www.stevebrownlee.com/alternative-teaching-software-development/)
```

C-Stein commented:
Incorporated Steve's changes and changed "thing" to "unique situation" at the end. "Second, teaching adults over the course of 6 months is a ~~thing~~ unique situation and the adults we accept into the program are expected to have the work ethic needed to forge their own path."
